src/components/MultiHopTrade/hooks/quoteValidation/useQuoteValidationErrors.tsx

```diff
@@ -64,21 +70,33 @@ export const useQuoteValidationErrors = (): ActiveQuoteStatus[] => {
     .minus(lastHopNetworkFeeCryptoPrecision)
     .gte(0)
 
-  const isBelowMinimumSellAmount = bnOrZero(sellAmountCryptoBaseUnit).lt(
-    minimumSellAmountBaseUnit ?? 0,
-  )
+  const isBelowMinimumSellAmount =
+    bnOrZero(sellAmountCryptoBaseUnit).gt(0) &&
+    bnOrZero(sellAmountCryptoBaseUnit).lt(minimumSellAmountBaseUnit ?? 0)
+
+  const feesExceedsSellAmount =
+    bnOrZero(sellAmountCryptoBaseUnit).isGreaterThan(0) &&
+    bnOrZero(buyAmountCryptoBaseUnit).isLessThanOrEqualTo(0)
+
+  /* TODO:
+    - no quotes available
+   */
 
   return [
-    !hasSufficientSellAssetBalance && ActiveQuoteStatus.InsufficientSellAssetBalance,
-    !firstHopHasSufficientBalanceForGas && ActiveQuoteStatus.InsufficientFirstHopFeeAssetBalance,
-    !lastHopHasSufficientBalanceForGas && ActiveQuoteStatus.InsufficientLastHopFeeAssetBalance,
-    isBelowMinimumSellAmount && ActiveQuoteStatus.SellAmountBelowMinimum,
+    !wallet && ActiveQuoteStatus.NoConnectedWallet,
     !walletSupportsSellAssetChain && ActiveQuoteStatus.SellAssetNotNotSupportedByWallet,
     !walletSupportsBuyAssetChain &&
       !manualReceiveAddress &&
       ActiveQuoteStatus.BuyAssetNotNotSupportedByWallet,
+    !hasSufficientSellAssetBalance && ActiveQuoteStatus.InsufficientSellAssetBalance,
+    !firstHopHasSufficientBalanceForGas && ActiveQuoteStatus.InsufficientFirstHopFeeAssetBalance,
+    !lastHopHasSufficientBalanceForGas && ActiveQuoteStatus.InsufficientLastHopFeeAssetBalance,
+    isBelowMinimumSellAmount && ActiveQuoteStatus.SellAmountBelowMinimum,
     !receiveAddress && ActiveQuoteStatus.NoReceiveAddress,
     !isTradingActiveOnSellPool && ActiveQuoteStatus.TradingInactiveOnSellChain,
     !isTradingActiveOnBuyPool && ActiveQuoteStatus.TradingInactiveOnBuyChain,
+    feesExceedsSellAmount && ActiveQuoteStatus.SellAmountBelowTradeFee,
+    insufficientBalanceProtocolFeeMeta && ActiveQuoteStatus.InsufficientFundsForProtocolFee,
+    quotes.length === 0 && ActiveQuoteStatus.NoQuotesAvailable,
   ].filter(isTruthy)
```

Comment:
would be cool to wrap all of this in a useMemo to prevent rerenders

Reply:
since creating a new array here results in recalc every frame

Reply:
gm https://github.com/shapeshift/web/pull/4917/commits/f7c30799409759f2a6041c6237bfc485a3b4e868